tests/fixtures/buffers.py

```diff
@@ -1,9 +1,25 @@
 import pytest
 from io import StringIO
+import os
+from pathlib import Path
+from tempfile import TemporaryDirectory
 
 
 @pytest.fixture
 def str_buff():
     """String IO fixture for simulating a file object"""
     with StringIO() as buffer:
         yield buffer
+
+
+@pytest.fixture
+def github_output(monkeypatch):
```

Comment:
you could use the `tmp_path` pytest fixture

Reply:
Fixed! Thanks :)